src/routes/sunshine.js

```diff
@@ -94,8 +95,54 @@ module.exports = (app, db) => {
         res.status(200).json(db.getFields());
     };
 
+    const exportHandler = async (req, res) => {
+        const query = queryFromReq(req);
+
+        // headers - array of strings
+        const headersToCSV = (headers) => {
```

Comment:
If you make use of the array functions `map` and `join` you can do this whole function in one line, or even just remove the function and do it directly below.
```JavaScript
const headersToCSV = (headers) => `${headers.map((s) => `"${s}"`).join(',')}\n`;
```